include/eve/module/core/constant/constant.hpp

```diff
@@ -53,13 +52,12 @@ namespace eve
 //! @}
 //================================================================================================
 
-template<typename T, auto BitsPattern>
-requires(plain_scalar_value<element_type_t<T>>)
+template<value T, auto BitsPattern>
 EVE_FORCEINLINE auto Constant(eve::as<T> const& = {})
```

Comment:
Why capital letter Comstant

Reply:
it's a old name from back there was a constant something. We never changed it. 
Do you want me to make it non capitalized and change evrywhere we may use it ?

Reply:
OK, they are only used in horner so, very few places